lib/tasks/general.rake

```diff
@@ -19,3 +19,15 @@ task fetch_all_user_bikes: :environment do
     FetchBikeIndexBikesJob.perform_later(user)
   end
 end
+
+task fetch_bikes_by_binx_id: :environment do
+  pp 'Batching all bicycles..'
```

Comment:
Using `pp` when you want to log things - particularly in production - isn't good form.

Switch to puts (or use the logger, but puts is fine here)